sdcm/nemesis.py

```diff
@@ -1337,56 +1333,61 @@ def _check_pod_stats(node):
         assert final_target_pod_stats
         assert final_target_pod_stats.get("restart_count") == origin_target_pod_stats.get("restart_count")
 
-    def disrupt_terminate_and_replace_node_kubernetes(self):  # pylint: disable=invalid-name
-        for node_terminate_method_name in self._get_kubernetes_node_break_methods():
-            self.set_target_node()
-            InfoEvent(f'OperatorNodeTerminateAndReplace ({node_terminate_method_name}) {self.target_node}').publish()
-            if not self._is_it_on_kubernetes():
-                raise UnsupportedNemesis('OperatorNodeTerminateAndReplace is supported only on kubernetes')
+    def disrupt_drain_kubernetes_node_then_replace_scylla_node(self):
+        self._disrupt_kubernetes_then_replace_scylla_node('drain_k8s_node')
 
-            self._disrupt_terminate_and_replace_node_kubernetes(self.target_node, node_terminate_method_name)
+    def disrupt_terminate_kubernetes_host_then_replace_scylla_node(self):
+        # NOTE: affected by the https://github.com/scylladb/scylla-operator/issues/643
```

Comment:
two issues referenced here, which one is the reason we are skipping ?

Reply:
The one used in the exception. Second appears also in case of decommission and it doesn't harm there.
So, the duplicated PVs bug is kind of cosmetic for now, as I can see.

Reply:
If it's harmless bug, let's drop this comment it doesn't help us.